src/main/java/mate/academy/dao/impl/UserDaoImpl.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.dao.impl;
+
+import org.hibernate.query.Query;
+import jakarta.persistence.criteria.CriteriaQuery;
+import mate.academy.dao.UserDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Movie;
```

Comment:
The import for `mate.academy.model.Movie` is unnecessary since this class deals with `User` entities. Consider removing it to clean up the imports.